src/main/scala/groundtest/Regression.scala

```diff
@@ -625,7 +625,7 @@ class MergedPutRegression(implicit p: Parameters) extends Regression()(p)
 
   io.cache.req.valid := (state === s_cache_req)
   io.cache.req.bits.cmd := M_XWR
-  io.cache.req.bits.typ := UInt(log2Ceil(64 / 8))
+  io.cache.req.bits.typ := MT_D
```

Comment:
This is the only use of MT_ outside of Rocket (MT_ is an ISA constant, not a TileLink constant).  Perhaps it should stay that way.